Problem while building

## Bug description

Has problem while building in `ledger live` project 
Link: https://github.com/zignartech/ledger-live/tree/LDGR-35

## Version

- Version: `2.0.0-rc.1`

## Hardware specification

- Operating system: `Pop!_OS 22.04 LTS x86_64`
- RAM: `23 GB`
- Number of cores: `16`
- Device type: `Predator PH315-54 V1.12`

## Steps To reproduce the bug

You should have `pnpm` installed at version `7.22.0`
1. git clone -b LDGR-35  https://github.com/zignartech/ledger-live.git
2. cd ledger-live
3. `pnpm i`
4. `pnpm build:lld`

## Expected behaviour

Should build.

## Actual behaviour

Throws error while building

## Errors

```js
ledger-live-desktop:build: ✘ [ERROR] Top-level await is not available in the configured target environment ("node12")
ledger-live-desktop:build: 
ledger-live-desktop:build:     ../../node_modules/.pnpm/@iota+iota.js@2.0.0-rc.1/node_modules/@iota/iota.js/dist/esm/polyfill-node.mjs:7:19:
ledger-live-desktop:build:       7 │     const fetch = (await import("node-fetch")).default;
ledger-live-desktop:build:         ╵                    ~~~~~
```
For this to work for me I must manually change the line 7 to this:

```js
const fetch = (import("node-fetch")).default.then();
```

--- a/packages/iota/src/polyfill-node.ts
+++ b/packages/iota/src/polyfill-node.ts
@@ -3,10 +3,13 @@
 
 // Fetch
 if (globalThis && !globalThis.fetch) {
-    // eslint-disable-next-line @typescript-eslint/no-require-imports,@typescript-eslint/no-var-requires
-    const fetch = require("node-fetch");
-    globalThis.Headers = fetch.Headers;
-    globalThis.Request = fetch.Request;
-    globalThis.Response = fetch.Response;
-    globalThis.fetch = fetch;
+    // eslint-disable-next-line unicorn/prefer-top-level-await, wrap-iife, no-void, func-names
+    void (async function() {
+        // eslint-disable-next-line @typescript-eslint/no-require-imports,@typescript-eslint/no-var-requires
+        const fetch = require("node-fetch");
+        globalThis.Headers = fetch.Headers;
+        globalThis.Request = fetch.Request;
+        globalThis.Response = fetch.Response;
+        globalThis.fetch = fetch;
+    })();
 }

